cmd/fsi.go

```diff
@@ -68,7 +69,22 @@ func (o *FSIOptions) Complete(f Factory, args []string) (err error) {
 
 // Link creates a FSI link
 func (o *FSIOptions) Link() (err error) {
-	return fmt.Errorf("not finished: link command")
+	pwd, err := os.Getwd()
```

Comment:
It's not a show stopper for me, but consider allowing that the directory be another command-line argument, and that it always be required. So linking the current directory would need to be done like `qri fsi link me/my_dataset .` with the "." at the end.

Also, there's also nothing to be gained by allowed an implied dataset ref to come from ".qri-ref", since we're creating a link. Even in the case that we want to re-link the current directory, we're never going to want to use the current value within ".qri-ref", since that would be a no-op.

Reply:
ah yes, I'd forgotten about this. I'll add it for `qri fsi link`, we should consider doing the same for `unlink` at some point in the future.